src/etc/config.xml.sample

```diff
@@ -323,6 +323,7 @@
     <enable/>
   </rrd>
   <ntpd>
+    <pool_server>0.opnsense.pool.ntp.org 1.opnsense.pool.ntp.org 2.opnsense.pool.ntp.org 3.opnsense.pool.ntp.org</pool_server>
```

Comment:
cleaning up the `timerserver` mess being anchored in `system` instead of `ntpd` would be much more appealing, but it requires MVC code to make this work appealing and future-proof.

Reply:
I mean the whole structure of the data is mind boggling. You want to create a server entry and add all options like also `pool_server` as  a property, but now you have all properties as global arrays that refer to `timeservers` which isn't even anchored here. This also negatively impacts partial config imports as it is BTW.

Reply:
Yes, the config is all over the place. I tried to "fit in" with the other options like `<prefer>` and just put the `<pool_server>` next to them. For this PR that seemed like the best thing to do, since my goal was to introduce the ability for pool configurations, not to refactor the entire thing.

Do you have alternative ideas where and how to store this data? One option that comes to mind would be to start the migration to a more sensible model and introduce a list of timeserver-tags that for now only contains a `<pool_server>[true|false]</pool_server>`, however the naming conventions for that structure should be agreed upon since they should last into the MVC age.